### DISCUSSION TITLE
AI based stack with pytest-plugin!

### DISCUSSION THREAD

**Discussion by Formartha** [pytest-dev/pytest#12090] [General]
Heya,

I'm currently developing a test stack that allows one to use test cases description to upload to an LM module and use pytest plugin to generate results.

I'm here to get ideas of development items, get your inputs about what is missing and invite you to join.
currently, it's under development on a beta phase - it works but there are limitations, so if you plan to integrate it in production environment, I suggest to wait at least a two weeks so I'd be able to close my backlog.

If you are a python developer, feel free to hop in, I'll be more than happy to explain how it works.

The repose are:
- pytest plugin: https://github.com/Formartha/pytest-ai1899
- the test stack: https://github.com/Formartha/ai1899

and if it's not to hard, please leave a star :), it helps a lot to know you did something good (or at least trying to)

**Comment by Formartha** [comment]
The project is up and running! the plugin interacts with the stack. Any use cases would be appreciated.

**Comment by RonnyPfannschmidt** [comment]
its completely and utterly unclear what this tool does or tries to accomplish

all i can see is a project shell that sends queries to a ai server that doesn't even get collection details and in the query, then returning a response followed by a name selection

without more context or a practical demo this looks very impractical and broken

**Reply by Formartha** [reply]
I guess you haven’t clicked on the ai1899 stack to read the description..
May I ask you to look at it and see if the description makes sense?

**Reply by RonnyPfannschmidt** [reply]
i took a look at the description, its a mix of marketing language that tells you nothing and a supposed image description with non-readable text

**Reply by Formartha** [reply]
ok, I accept your inputs, however I'm trying to translate it into some functional AIs.
Would you like to see examples? e.g. payload and reference for such a collection? there is a folder with json as an example for multi upsert collection.

What I'm going to add is also high level design that show the stack and the data flow, this might give more clarity.
